docs/src/content/docs/auth/proofkit.mdx

```diff
@@ -69,25 +69,25 @@ From [The Copenhagen Book](https://thecopenhagenbook.com/sessions):
 
 The session layout also includes related fields for the user, so user info can be fetched in a single request via the FileMaker Data API. **Unless you change the types in the code**, you should expect all data from this layout to be available to the currently logged in user in the browser.
 
-By default, the layout for the session table should be named `proofkit_auth_sessions` but this can be customized in the fmschema.config.mjs file.
+By default, the layout for the session table is named `proofkit_auth_sessions` but this can be customized in the fmschema.config.mjs file.
```

Comment:
Can we re-think this wording? I want to make it clear that the layouts can be renamed if you edit the fmscheama.config file, but that these are default expected names if you don't use the add-on for some reason and just make the layouts manually

Reply:
using "should be" seems to imply the default naming might fail. That's why I suggested "is"